ivy/functional/frontends/jax/numpy/name_space_functions.py

```diff
@@ -116,3 +116,7 @@ def var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False, *, where=Non
     if ivy.is_array(where):
         ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
     return ret.astype(dtype)
+
+
+def amin(a, axis=None, out=None, keepdims=None, initial=None, where=None):
+    return ivy.min(a, axis=axis, out=out, keepdims=keepdims)
```

Comment:
Could you please add functionality to emulate the `where` and `initial` parameters as well? 

The `where` parameter can be handled using `ivy.where`, see examples in this file here on more details of how to implement this, or let me know if you need help! :)

Reply:
See also [here](https://numpy.org/doc/stable/reference/generated/numpy.ufunc.reduce.html#numpy.ufunc.reduce) and [here](https://numpy.org/doc/stable/reference/generated/numpy.amin.html) for more info on the `initial` parameter.

Reply:
the numpy frontends [`sum`](https://github.com/unifyai/ivy/blob/e02ae8eb46a20bcda72688fb3ac08080a33562f5/ivy/functional/frontends/numpy/mathematical_functions/sums_products_differences.py#L9) might also provide some guidance on how to handle the `initial` argument.